webpack.config.production.js

```diff
@@ -1,23 +1,23 @@
 /* eslint-disable */
 
-var path = require("path");
-var webpack = require("webpack");
+var path = require('path');
```

Comment:
@knowbody -- This should be linted as node code, not frontend React code. We should use https://github.com/walmartlabs/eslint-config-defaults/blob/master/configurations/walmart/es5-node.js for node files.